itemtemplates.md

```diff
@@ -0,0 +1,67 @@
+# Items Endpoints
```

Comment:
This header should make it clear that these Endpoints are related to Collection Item Templates.

Reply:
Good catch, I've updated the header